rtgui/previewhandler.cc

```diff
@@ -204,27 +204,19 @@ Glib::RefPtr<Gdk::Pixbuf> PreviewHandler::getRoughImage (int x, int y, int w, in
             h = image->getHeight() * totalZoom;
         }
 
-        int ix = x * zoom;
-        int iy = y * zoom;
+        x *= zoom;
+        y *= zoom;
 
-        if (ix < 0) {
```

Comment:
Don't we need to check for `x < 0` and `y < 0` anymore? `x = std::max(x, 0)` and `y = std::max(y, 0)` should do the trick.